nuget/Auth0.OidcClient.WPF.nuspec

```diff
@@ -130,13 +133,19 @@
         <dependency id="Auth0.OidcClient.Core" version="3.4.1" />
         <dependency id="Microsoft.Toolkit.Wpf.UI.Controls.WebView" version="6.1.1"/>
       </group>
+      <group targetFramework="net6.0-windows7.0">
```

Comment:
Why do we need `windows7.0` here?

Reply:
Or else you will get the [NU1012 error](https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1012).

Reply:
The truth is that there is little information in the documentation. I guess `windows7.0` indicates the minimum platform.

If you activate the option _Project properties/Package/General/Generate NuGet package on build_, Visual Studio will create a NuGet package with the information of the project file itself (without taking into account the _.nuspec_ file in the directory _NuGet Packaging_) and the file created is the following, as you can see it does it with the postfix `windows7.0`:

```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
  <metadata>
    <id>Auth0.OidcClient</id>
    <version>1.0.0</version>
    <authors>Auth0.OidcClient</authors>
    <owners></owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <description>Package Description</description>
    <dependencies>
      <group targetFramework=".NETCoreApp3.1">
        <dependency id="Auth0.OidcClient.Core" version="3.4.1" exclude="Build,Analyzers" />
        <dependency id="IdentityModel.OidcClient" version="5.2.1" exclude="Build,Analyzers" />
        <dependency id="Microsoft.CSharp" version="4.7.0" exclude="Build,Analyzers" />
        <dependency id="Microsoft.Toolkit.Wpf.UI.Controls.WebView" version="6.1.2" exclude="Build,Analyzers" />
      </group>
      <group targetFramework="net6.0-windows7.0">
        <dependency id="Auth0.OidcClient.Core" version="3.4.1" exclude="Build,Analyzers" />
        <dependency id="IdentityModel.OidcClient" version="5.2.1" exclude="Build,Analyzers" />
        <dependency id="Microsoft.Web.WebView2" version="1.0.1823.32" exclude="Build,Analyzers" />
      </group>
      <group targetFramework=".NETFramework4.6.2">
        <dependency id="Auth0.OidcClient.Core" version="3.4.1" exclude="Build,Analyzers" />
        <dependency id="IdentityModel.OidcClient" version="5.2.1" exclude="Build,Analyzers" />
        <dependency id="Microsoft.CSharp" version="4.7.0" exclude="Build,Analyzers" />
        <dependency id="Microsoft.Toolkit.Wpf.UI.Controls.WebView" version="6.1.2" exclude="Build,Analyzers" />
      </group>
    </dependencies>
    <frameworkReferences>
      <group targetFramework=".NETCoreApp3.1">
        <frameworkReference name="Microsoft.WindowsDesktop.App.WPF" />
      </group>
      <group targetFramework="net6.0-windows7.0">
        <frameworkReference name="Microsoft.WindowsDesktop.App.WPF" />
      </group>
      <group targetFramework=".NETFramework4.6.2" />
    </frameworkReferences>
  </metadata>
</package>
```